blemmy/home/models.py

```diff
@@ -15,88 +15,93 @@
 from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
 from wagtail.wagtailsearch import index
 
-class ArticleIndexPage(Page):
```

Comment:
I don't think we should remove these models from the home app. Furthermore, commenting-out is not a good way to do this.